app/f0_serialcom/main.c

```diff
@@ -0,0 +1,97 @@
+/* STM32F0xx Serial terminal example
+ *
+ * Jeff Ciesielski <jeff.ciesielski@gmail.com>
+ */
+
+/* NOTE: This example assumes that you are attaching a serial
+ * terminal to USART1 (PA9/PA10) via something like an FTDI cable */
+
+#include <FreeRTOS.h>
+#include <task.h>
+
+#include <stm32f0xx_misc.h>
+#include <blinky.h>
+
+#include <stdio.h>
+#include <usart.h>
+#include <shell.h>
+#include <alloca.h>
+
+void vApplicationTickHook(void)
+{
+}
+
+void vApplicationStackOverflowHook(xTaskHandle pxTask, signed char *pcTaskName)
```

Comment:
Are you sure about the type of pcTaskName? I know the type is mandated by FreeRTOS, but could you double check that it's really a signed char pointer? I'd have expected a bare-word char...